packages/now-cli/src/commands/domains/rm.ts

```diff
@@ -85,6 +86,18 @@ export default async function rm(
     return 1;
   }
 
+  const projects = await findProjectsForDomain(client, domain.name);
+
+  if (Array.isArray(projects) && projects.length > 0) {
+    output.warn(
+      `The domain is currently used by ${plural(
```

Comment:
It may be nice to list out the specific project+domain that will be affected when removing the apex domain as well, to make sure whoever entered the command has a clear idea of whats going to happen when they remove their apex domain.

Reply:
Made sure we show the projects, wouldn't the domain be redundanten information?

Reply:
Maybe, but removing `apex.com` could remove a few subdomains for example:
```
vercel domains rm apex.com

- apex.com
- api.apex.com
- app.apex.com
```

which may not be clear if we just list the project name (cc @leo)

Reply:
Unfortunately the ping doesn't contain much context about what's going on here, but I read the last few messages in the thread and it seems to be about rendering enough useful information when removing a Domain.

I don't think we can list all the occurances, since that could potentially not fit in the screen. I'd be as verbose as possible, but only list the count (and I think we're already doing that).